### PR TITLE
GUNCRAFTING 4: OLD MAN WINTER

### PR DESCRIPTION
**Pull request by Iamgoofball**
original PR with the old comments located at #19250, opened this one due to the mass amount of outdated review stuff lagging down browsers

dreamseeker_2016-12-29_12-30-28

# HOW TO QUICKLY ASSEMBLE A GUN:
1. PRINT FRAME, **PROJECTILE** OR **ENERGY**
2. ATTACH A **BARREL**, **HANDLE**, AND **BASE**
3. **OPTIONAL**: ATTACH A **SCOPE**, **UNDERBARREL**, AND **BULLET MODIFICATION**
4. **SHOOT SHIT**

TODO:
- [x] Attachment Bases
- [x] Handles
- [x] Bases
- [x] Barrels
- [x] Frames
- [x] Attachments(1st Wave)
- [x] Way to get parts
- [x] Codersprites

STAGES:
- [x] Proof of Concept
- [x] Feature Fulfillment/Testing
- [x] Cleanup
- [x] SPRITES (thanks bman)

### CODE REVIEW

**Comment by Militaires** [comment]
current OP sprites not up to date before you talk shit kebab

**Comment by dannno** [comment]
are you doing it so that it's all just 1:1 with current guns

**Comment by Iamgoofball** [comment]
i'm working on it

**Comment by dannno** [comment]
cool, good luck hope it works

File: _maps/map_files/Deltastation/DeltaStation2.dmm [hunk 1]
```diff
@@ -116882,7 +116843,31 @@
...
+	d1 = 2;
+	d2 = 8;
+	icon_state = "2-8"
+	},
+/obj/structure/lattice/catwalk,
+/turf/open/space,
+/area/solar/starboard)
+"ebF" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plasteel/warning/end{
```
Comment (Iamgoofball) [inline _maps/map_files/Deltastation/DeltaStation2.dmm]: crap was I on an old copy

**Comment by Supermichael777** [comment]
you could alter news to auto-replace.

**Comment by Iamgoofball** [comment]
?????

**Comment by optimumtact** [comment]
not a fix or feature

**Comment by Iamgoofball** [comment]
Grandfathered in pre freeze

**Pull request closed without merging**